js/common.js

```diff
@@ -9,8 +9,13 @@ var firstScriptTag = document.getElementsByTagName('script')[0];
 firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
 
 // Array of videoIds
-// The key corresponds to the data attributes in about/index.html
+// The key corresponds to the data attributes in docs/talks/index.html
+// TODO(jtattermusch): is there a way to not require editing this when adding new videos?
 var playerInfoList = [
+    {type: 'yt', key: '5QH67PXNbaI'},
```

Comment:
btw, I think requiring this is pretty unfortunate, it makes it very hard to add new videos.